cli/tests/pcluster/config/test_section_cluster.py

```diff
@@ -825,10 +825,7 @@ def test_cluster_from_file_to_cfn(mocker, pcluster_config_reader, settings_label
         "pcluster.config.param_types.get_avail_zone",
         side_effect=lambda subnet: "mocked_avail_zone" if subnet == "subnet-12345678" else "some_other_az",
     )
-    mocker.patch(
-        "pcluster.config.validators.get_supported_features",
-        return_value={"instances": ["t2.large"], "baseos": ["ubuntu1804"], "schedulers": ["slurm"]},
-    )
+    mocker.patch("pcluster.config.validators.get_efa_enabled_instance_types", return_value=["t2.large"])
```

Comment:
can we use the boto3_stubber in the tests to mock the call to describe_instance_types so that we perform some extra validations?